src/cljs/rems/application.cljs

```diff
@@ -362,6 +362,10 @@
                          :accepted (contains? accepted-licenses (:license/id license))
                          :readonly readonly?)
                   show-accepted-licenses?]))
+         (when (contains? permissions :application.command/add-licenses)
```

Comment:
This assumes that the add-licences and accept-licenses functionality are never enabled for the same user, which is fine but it's a silent assumption.

Reply:
Good point. This is what it looks like when both are enabled:
![add-and-accept](https://user-images.githubusercontent.com/207385/65943074-4e546a00-e437-11e9-9881-255d785af9ec.png)